src/lib/db/access-store.ts

```diff
@@ -138,18 +217,28 @@ export class AccessStore implements IAccessStore {
         return rows.map((r) => r.user_id);
     }
 
-    async addUserToRole(userId: number, roleId: number): Promise<void> {
+    async addUserToRole(
+        userId: number,
+        roleId: number,
+        projecId: string,
```

Comment:
Should this be optional?

Reply:
Yes, if not it should be renamed to `addUserToProjectRole`.

Reply:
Yeah, it probably should, you are correct. Updated to optional